reana/reana_dev/cluster.py

```diff
@@ -236,7 +236,7 @@ def cluster_build(
     "--workspace-mounts",
     multiple=True,
     help="Which directories from the Kubernetes nodes to mount inside the cluster pods? "
-    "cluster_node_path:job_pod_mountpath, e.g /var/reana/mydata:/mydata",
+    "cluster_node_path:cluster_pod_mountpath, e.g /var/reana/mydata:/mydata",
```

Comment:
Can you explain why this was needed? `/mydata` is `job_pod_mountpath`

Reply:
But this paths are mounted not only for the jobs, but also for cluster pods `r-w-controller` and `reana-server`. I thought that `job_pod_mountpath` was not completely correct (but cluster_pod_mountpath might also not be correct, as is not for all cluster pods.....).